### ISSUE TITLE
How to ensure checkpoint is loaded

### ISSUE THREAD

**Issue by chenwydj** [facebookresearch/detectron2#1007]
## ❓ How to ensure checkpoint is loaded

I can see the following logs during loading the checkpoint.

Does that mean that any model parameters not mentioned below are satisfied and successfully loaded?

Thank you!

```
[03/05 20:00:13] fvcore.common.checkpoint INFO: Some model parameters are not in the checkpoint:
  proposal_generator.anchor_generator.cell_anchors.0
  proposal_generator.rpn_head.conv.{weight, bias}
  proposal_generator.rpn_head.objectness_logits.{weight, bias}
  proposal_generator.rpn_head.anchor_deltas.{weight, bias}
  roi_heads.box_predictor.cls_score.{weight, bias}
  roi_heads.box_predictor.bbox_pred.{weight, bias}
```

**Comment by ppwwyyxx** [comment]
yes

**Comment by chenwydj** [comment]
Thank you @ppwwyyxx !

Could you point me the repo by which the ImageNet pretrained model is trained?
For example: detectron2://ImageNetPretrained/MSRA/R-50.pkl

**Comment by chenwydj** [comment]
@ppwwyyxx 

Currently, I managed to successfully load the pretrained model provided by [NPID](https://github.com/zhirongw/lemniscate.pytorch#pretrained-model), which is also used by Dr. Ishan Misra in their [Table 1](https://arxiv.org/pdf/1912.01991.pdf).

However, the default (batchsize=16, LR=0.02) setting for Faster-RCNN VOC detection will make the bbox regression L1 loss NAN in the first few epochs during training. This can be solved by using a much smaller LR (like 0.1 smaller), but will give inferior performance.

Could you give some suggestions on why the bbox regression L1 loss becomes NAN?

Thank you!

```
Traceback (most recent call last):
  File "/home/chenwy/detectron2/detectron2/engine/train_loop.py", line 132, in train
    self.run_step()
  File "/home/chenwy/detectron2/detectron2/engine/train_loop.py", line 217, in run_step
    self._detect_anomaly(losses, loss_dict)
  File "/home/chenwy/detectron2/detectron2/engine/train_loop.py", line 240, in _detect_anomaly
    self.iter, loss_dict
FloatingPointError: Loss became infinite or NaN at iteration=18!
loss_dict = {'loss_cls': tensor(nan, device='cuda:0', grad_fn=<NllLossBackward>), 'loss_box_reg': tensor(nan, device='cuda:0', grad_fn=<DivBackward0>), 'loss_rpn_cls': tensor(0.6930, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rpn_loc': tensor(0.2337, device='cuda:0', grad_fn=<MulBackward0>)}
```

**Comment by ppwwyyxx** [comment]
The source of pre-trained models are explained in the model zoo.

As the [issue template](https://github.com/facebookresearch/detectron2/blob/master/.github/ISSUE_TEMPLATE/unexpected-problems-bugs.md) mentions:

> If you expect the model to converge / work better, note that we do not give suggestions on how to train a new model. Only in one of the two conditions we will help with it: (1) You're unable to reproduce the results in detectron2 model zoo. (2) It indicates a detectron2 bug.
